extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -111,6 +111,7 @@ export let InteractiveReactData;
  *    mutedState: boolean,
  *    pageAudioState: boolean,
  *    pageHasElementsWithPlaybackState: boolean,
+ *    panningMediaState: {x: number, y: number, zoom: number},
```

Comment:
I think we need to write a comparison function in the store for this state, right?

Reply:
Since I'm creating a new object with each `SET_PANNING_MEDIA_STATE`, `oldState[key]` would never be the same object as `this.state_[key]` in `dispatch`. 

Why is the comparison necessary if I'm not sending a mutated object that risks having the same reference?

Reply:
I assume that when you introduce linking elements via an attribute (so you transform this to a dictionary), then comparing this will be crucial. For now it's not necessary and since you'll write one when necessary, you should be able to use the default compare function. 👍